src/NativeOpentokReactNative.ts

```diff
@@ -122,6 +136,11 @@ export interface Spec extends TurboModule {
   getSubscriberRtcStatsReport(): void;
   publish(publisherId: string): void;
   sendSignal(sessionId: string, type: string, data: string): void;
+  setEncryptionSecret(sessionId: string, secret: string): Promise<void>;
+  reportIssue(sessionId: string): Promise<string>;
+  forceMuteAll(sessionId: string, excludedStreamIds: string[]): Promise<boolean>;
+  forceMuteStream(sessionId: string, streamId: string): Promise<boolean>;
+  disableForceMute(sessionId: string): Promise<boolean>;
```

Comment:
These aren't implemented in the Android side, so it breaks the Android build. I'll implement them in Android and add a commit to the PR.